core/src/apps/management/reset_device/layout.py

```diff
@@ -7,7 +7,7 @@
 from trezor.ui.checklist import Checklist
 from trezor.ui.info import InfoConfirm
 from trezor.ui.scroll import Paginated
-from trezor.ui.shamir import NumInput
+from trezor.ui.slip39 import NumInput
```

Comment:
Actually, shouldn't we call it just `num_input.py`? We might use it somewhere else later?

Reply:
ok, I renamed it.